README.md

```diff
@@ -1,167 +1,29 @@
-# Prem Operator
+## Prem Operator: Advanced AI Model Deployment on Kubernetes
 
-Deploy AI models to Kubernetes using LocalAI, vLLM, DeepSpeed-MII, NVIDIA Triton and custom Docker images.
```

Comment:
I'd like to keep a one line description which specifies what the operator does with the minimum number of words. We could maybe add "easily" at the start?

Reply:
Can you suggest exact sentence you would like to see?

Reply:
That sentence with "Easily" on the front would be fine IMO

Reply:
Done